components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -202,10 +202,23 @@ export class Panel extends React.Component<Props, State> {
             break
           }
 
-          if (notification.args[1] === '0') {
-            text = getMessage('contributeNotificationSuccess')
-          } else {
-            text = getMessage('contributeNotificationError')
+          const result = notification.args[1]
+
+          // Results
+          // 0 - success
+          // 1 - general error
+          // 14 - auto contribution table is empty
+          // 15 - not enough funds
+          // 16 - error while tipping
+
+          if (result === '0') {
+            const fixed = utils.convertProbiToFixed(notification.args[3])
+            text = getMessage('contributeNotificationSuccess',
+              [fixed])
+          } else if (result === '14') {
+            text = getMessage('contributeNotificationEmptyTable')
+          } else if (result === '15') {
+            text = getMessage('contributeNotificationNotEnoughFunds')
           }
```

Comment:
Why don't we handle all available errors here from the table? Or at least have an else clause for if the error is outside of those.